Client/Frontend/Browser/Browser.swift

```diff
@@ -117,6 +117,8 @@ class Browser: NSObject, BrowserWebViewDelegate {
             configuration!.userContentController = WKUserContentController()
             configuration!.preferences = WKPreferences()
             configuration!.preferences.javaScriptCanOpenWindowsAutomatically = false
+            configuration!.allowsInlineMediaPlayback = true
+            configuration!.mediaPlaybackAllowsAirPlay = true
```

Comment:
Was the intention to also enable Airplay? I think it's a good idea - I don't see why we wouldn't but it wasn't mentioned in the bug. If you did, you'll want to use the `allowsAirPlayForMediaPlayback` property instead because `mediaPlaybackAllowsAirPlay` is deprecated in 9.0. You can use Swift's conditional ifs to use the correct one like so:

```swift
if #available(iOS 9, *) {
    allowsAirPlayForMediaPlayback
} else {
    mediaPlaybackAllowsAirPlay
}
```